[s] Lavaland Patch

Xhuis opened the pull request:
Keeping [s] in the title to avoid spoilers, as lavaland is called "unexplored" for a reason.
- Massively modified the flavor text, names, descriptions, and appearances of all ghost role spawners. These include stranded animal hospital vets, surviving prisoners, stranded hermits, ash walkers, lifebringers (plantmen), free golems of both kinds, and wish granter exiles. These have also all been moved to `ghost_role_spawners.dm`.
- The necropolis gate can be knocked on multiple times, but every time after the first results in nobody answering the door. There's also now a delay between the knock and the actual spawn of Legion.
- [Legion has a new spawn sound and effect.](https://dl.dropboxusercontent.com/u/116194515/legion_spawn.mp4) Both the color change and the sound effect are played across the entire z-level that Legion spawns on.
- Goliaths and watchers now have more ambiguous descriptions.
- Ash storms and cursed slot machines now have sounds.

**Token: #18387**

Commit:
Commit squash before update to master

ChangelingRain commented on code/game/objects/structures/ghost_role_spawners.dm:
```diff
@@ -0,0 +1,240 @@
+	var/obj/structure/S = new(get_turf(src))
+	S.name = "empty terrarium"
+	S.desc = "An ancient machine that seems to be used for storing plant matter. Its hatch is ajar."
+	S.icon = 'icons/obj/lavaland/spawners.dmi'
```
make a subtype structure please

Xhuis replied:
I figured that I shouldn't bother with something that'll only be used once... but then I remembered the fallen ones.
The chronosuit.
The liberation station.
I shall do as you ask.

Cheridan commented:
holy shit

Man-Shroom- commented:
OH SHIT LORE

**I AM SUMMONED**

Xhuis commented:
_New in this commit:_
* Added fluff structures, which are used only for enriching the environment. They can be disassembled into a single piece of metal with a wrench.
* Refactored the empty terrarium into a fluff structure.

Man-Shroom- commented on code/game/objects/structures/ghost_role_spawners.dm:
```diff
@@ -0,0 +1,236 @@
+
+/obj/effect/mob_spawn/human/prisoner_transport/New()
+	var/list/crimes = list("murder, but whatever they might say, you <i>know</i> that it was self-defense", \
+	"rape, but they're lying since you both agreed first", \
```
>rape

Xhuis replied:
I tried to be realistic. If a server that's 18+ can't handle it, then I can remove it.

KorPhaeron replied:
No rape

Man-Shroom- commented on code/modules/ruins/lavaland_ruin_code.dm:
```diff
@@ -188,251 +194,3 @@
-		new /obj/effect/mob_spawn/human/ash_walker(get_step(src.loc, SOUTH))
-		visible_message("<span class='danger'>An egg is ready to hatch!</span>")
-		meat_counter -= 2
-
```
Everything above in this particular .dm looks important and i don't think the ash walker tendril is defined elsewhere

Xhuis replied:
That's odd. I'll readd that.

Man-Shroom- commented on code/game/objects/structures/ghost_role_spawners.dm:
```diff
@@ -0,0 +1,236 @@
+	mob_species = /datum/species/pod
+	flavour_text = "<font size=3><b>Y</b></font><b>ou are a sentient ecosystem - an example of the mastery over life that your creators possessed. Your masters, benevolent as they were, created uncounted \
+	seed vaults and spread them across the universe to every planet they could chart. You are in one such seed vault. Your goal is to cultivate and spread life wherever it will go while waiting \
+	for contact from your creators. Estimated time of last contact: Deployment, 5x10^3 millennia ago.</b>"
```
That's 5,000 years which means lavaland predates humans by a bit

Xhuis replied:
It's a planet, and the Necropolis is a fallen civilization, so I figured it'd be a while.

Man-Shroom- replied:
good good good this means that i can check off the 'lavaland is the remains of clown planet' box from my big theory wall

coiax replied:
A millennia is a 1000 years, so  this is 5,000,000 years ago.

Man-Shroom- commented on code/game/objects/structures/ghost_role_spawners.dm:
```diff
@@ -0,0 +1,236 @@
+	death = FALSE
+	random = TRUE
+	mob_species = /datum/species/human
+	flavour_text = "<font size=3><b>Y</b></font><b>ou've been stranded in godless prison of a planet for longer than you can remember. Each day you barely scrape by, and between the terrible \
```
Stranded in THIS godless prison of a planet

Xhuis replied:
Thanks, didn't spot that.

Man-Shroom- commented on code/game/objects/structures/ghost_role_spawners.dm:
```diff
@@ -0,0 +1,236 @@
+	"grand theft, but the spaceship was good-looking and just <i>sitting there</i> and idling", \
+	"collaboration with the Syndicate, but your pay is terrible and theirs was lucrative", \
+	"worship of a forbidden deity, but it wasn't hurting anyone and you know that they're just xenophobic", \
+	"interracial relations, but as far as you know, there isn't a law against that", \
```
interspecies relations

Xhuis replied:
Merci

Xhuis commented:
_New in these commits:_
* Added fluff structures for all lavaland ghost roles that were missing them.
* Made prisoner containment sleepers use the Syndicate color scheme.
* Re-added the ash walker nest that was removed accidentally.
* Fixed some improper grammar.

Xhuis commented:
_New in this commit:_
* [Added a colossus arena, slightly too large to picture fully](https://dl.dropboxusercontent.com/u/116194515/arena.png). The rune in the center, when invoked by someone who stands still for a total of ten seconds, will spawn a colossus. Colossi can also no longer spawn naturally. The arena, however, gives off a Distorted Signal and attempts to spawn every round if possible.
* The disintegration ball fired from the colossus now moves at a rate that can be outrun rather than resulting in an instant death.
* Watchers now have a five-second ranged shot cooldown (up from three seconds) and no longer fire freezing rays, instead firing "teardrops" that do 15 burn damage.
* Tweaked the descriptions of goliaths and watchers to be more ambiguous as to the nature of the beasts.

Xhuis commented:
_New in this commit:_
* Added sounds for ash storms and framework for sounds for other weather types.
 * [Windup ](https://dl.dropboxusercontent.com/u/116194515/ash_storm_windup.ogg)| [Start](https://dl.dropboxusercontent.com/u/116194515/ash_storm_start.ogg) | [End](https://dl.dropboxusercontent.com/u/116194515/ash_storm_end.ogg)
* Changed the messages of weather to be more obvious.
* Tweaked a lot of grammar for a few different mobs and objects.
* Added a 10% chance for any ash storm to be too far off to deal any real damage, instead resulting in a light and harmless ashfall.

KorPhaeron commented:
Did you use cuboos's sounds?

Man-Shroom- commented on code/modules/ruins/lavaland_ruin_code.dm:
```diff
@@ -188,251 +304,3 @@
-	..()
-	var/area/A = get_area(src)
-	if(A)
-		notify_ghosts("An ash walker egg is ready to hatch in \the [A.name].", source = src, action=NOTIFY_ATTACK)
```
This tidbit seems important too

How did you accidentally wipe this stuff?

Xhuis replied:
This still exists, check `ghost_role_spawners`.

Man-Shroom- commented:
>ROT1 words and a khornate symbol

nice nice nice

xxalpha commented:
I am pretty sure WJ was remaking lavaland.

KorPhaeron commented on code/game/objects/structures/ghost_role_spawners.dm:
```diff
@@ -0,0 +1,246 @@
+	"larceny, but they looked loaded and you know that they could have just bought another one", \
+	"embezzlement, but there was plenty to go around and you know that it was barely a dent", \
+	"dereliction of duty, but you'd been having a rough day and you thought you might have caught Pierrot's throat", \
+	"kidnapping, but they were a Syndicate spy and you had conclusive evidence about it had they bothered to ask", \
```
I feel like some of these are a bit specific and don't leave as much room for players to make things up

Xhuis replied:
I can make 'em more vague

KorPhaeron commented on code/modules/mob/living/simple_animal/hostile/mining_mobs.dm:
```diff
@@ -721,20 +721,30 @@
 	stat_attack = 1
 	robust_searching = 1
 	loot = list()
+	ranged_cooldown_time = 50
```
hugbox

Xhuis replied:
:(

KorPhaeron commented on code/modules/ruins/lavaland_ruin_code.dm:
```diff
@@ -1,31 +1,51 @@
 	burn_state = LAVA_PROOF
 	luminosity = 1
 	var/boss = FALSE
+	var/is_anyone_home = FALSE
```
Why a second var to do the same thing?

Xhuis replied:
This is used for the "knocking on the door" fluff to prevent people from spamming the knock over and over.

KorPhaeron commented on code/game/objects/structures/ghost_role_spawners.dm:
```diff
@@ -0,0 +1,246 @@
+	anchored = 0
+	density = 0
+	flavour_text = "<font size=3><b>Y</b></font><b>ou are an ash walker. Your tribe worships <span class='danger'>the Necropolis</span>. The wastes are sacred ground, its monsters a blessed bounty. \
+	You have seen lights in the distance... they foreshadow the arrival of outsiders that seek to tear apart the Necropolis and its domain. Fresh sacrifices for the Dark One.</b>"
```
Who is "The Dark One"

coiax replied:
Order of the Stick reference I think

Xhuis replied:
Nope, the Dark One is Legion.

KorPhaeron replied:
Legion isnt even the boss of lavaland just you wait

KorPhaeron replied:
Please remove reference to "The Dark One"

Xhuis replied:
He's not? Oh god. I'll remove the references though.

KorPhaeron commented:
This is a lot of stuff to put in one PR. I liked all the fluff changes but now you're going into balance stuff

KorPhaeron commented:
You have not reworked watchers so much as made them incredibly weak

KorPhaeron commented on code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm:
```diff
@@ -4,7 +4,7 @@
 	icon_state = "legion"
 	icon_living = "legion"
-	desc = "One of many."
+	desc = "One of many. One of many. One of many. One of many. One of many."
```
Seems excessive

KorPhaeron commented on code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm:
```diff
@@ -32,6 +32,13 @@
 /mob/living/simple_animal/hostile/megafauna/legion/New()
 	..()
 	new/obj/item/device/gps/internal/legion(src)
+	for(var/mob/M in mob_list - src)
```
in player_list

ChangelingRain commented on code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm:
```diff
@@ -32,6 +32,13 @@
+		if(M.z == z)
+			M << "<span class='userdanger'>Discordant whispers flood your mind in a thousand voices. Each one speaks your name, over and over. Something horrible has come.</span>"
+			M << 'sound/creatures/legion_spawn.ogg'
+			if(M.client)
```
instead of these three lines, `flash_color(M, color="#FF0000", time=50)`

KorPhaeron commented:
I would like to be the one to merge this in the end

coiax commented on code/modules/ruins/lavaland_ruin_code.dm:
```diff
@@ -76,19 +167,44 @@
+
+/mob/living/simple_animal/hostile/spawner/ash_walker/proc/consume()
+	for(var/mob/living/H in view(src,1)) //Only for corpse right next to/on same tile
+		if(H.stat)
```
H.stat == DEAD, assuming it's numeric is the DEVIL

xxalpha replied:
This holy shit, the defines exist for a reason.

Xhuis replied:
I figured that unconsciousness worked to consume them, as it's been a regular tactic of ash walkers to feed critted brethren to the tendril

coiax commented on code/modules/ruins/lavaland_ruin_code.dm:
```diff
@@ -76,19 +167,44 @@
+	if(meat_counter >= 2)
+		new /obj/effect/mob_spawn/human/ash_walker(get_step(src.loc, SOUTH))
+		visible_message("<span class='danger'>One of the eggs swells to an unnatural size and tumbles free. It's ready to hatch!</span>")
+		meat_counter -= 2
```
replace 2 with a DEFINE in the file, so we can later change it if needed

coiax commented on code/modules/mining/lavaland/weather_types.dm:
```diff
@@ -88,8 +91,8 @@
 	duration_overlay = "light_ash"
-	duration_message = "Looks like the storm just missed the mining area. False alarm."
-	wind_down_message = "The ash fall starts to trail off."
+	duration_message = "<span class='notice'>Gentle ashfall surrounds you like grotesque snow. The storm seems to have passed you by.</span>"
```
ash, snow but SUPER UGLY

KorPhaeron replied:
Some of these descriptions seem a bit flowery for stuff you're gonna have in big red text every couple minutes

Xhuis replied:
I like writing descriptive text. If it's that bad, I can change it.

KorPhaeron replied:
I'm undecided on this. Lavaland is already pretty over the top, but my descriptions are admittedly lazy

MMMiracles commented on code/game/objects/structures/ghost_role_spawners.dm:
```diff
@@ -0,0 +1,246 @@
+			back = /obj/item/weapon/storage/backpack
+		if(3)
+			flavour_text += "you were a doctor on one of Nanotrasen's space stations, but you left behind that damn corporation's tyranny and everything it stood for. From a metaphorical hell \
+			to a little one, you find yourself nonetheless missing the recycled air and warm floors of what you left behind... but you'd still rather be here than there.</b>"
```
*literal

Xhuis replied:
Whoops, misspelled

coiax commented:
Where are the specialised sleepers for the cancer rat and Bessie? Also, while you're tweaking that, maybe make it so they can open their doors or at least notify the vets that the patients are awake. Seeing a suicided cancer rat is saddening.

KorPhaeron commented on code/modules/ruins/lavaland_ruin_code.dm:
```diff
@@ -36,6 +56,73 @@
+	name = "sanguine sigil"
+	desc = "A sickening glyph drawn amongst the ashes in fresh blood."
+	icon = 'icons/effects/96x96.dmi'
+	icon_state = "landing"
```
Don't think you should reuse the "a boss is attacking please move" rune with something you are supposed to click.

Also a rune laying in an empty area is a bit dull

Xhuis replied:
I'm all ears for more thematic suggestions.

KorPhaeron commented on code/modules/ruins/lavaland_ruin_code.dm:
```diff
@@ -36,6 +56,73 @@
+		var/words = "Ayy lmao"
+		switch(i)
+			if(1)
+				words = "H rtllnm xnt, fnn-zq... fzqczxd... ftzqchzm ne sghr Mdbqnonkhr..."
```
What do these translate as?

MMMiracles replied:
ROT1 memes

I summon you, goo-ar... gardaye... guardian of this Necropolis...

KorPhaeron replied:
fucking clock cult trying to infect lavaland

Xhuis replied:
It's a challenge issued to the colossus, run through ROT25.

Iamgoofball commented on code/game/objects/structures/fluff.dm:
```diff
@@ -0,0 +1,43 @@
+
+/obj/structure/fluff
+	name = "fluff structure"
+	desc = "Fluffier than a sheep. This shouldn't exist."
```
needs a meme icon

Xhuis replied:
You're not my real dad

JJRcop replied:
yeah why make it an abstract path that should never be used. why not take the time to shoehorn in a dumb story!?

Xhuis replied:
You're not my real dad

Dorsisdwarf commented:
:-1: to the collossus trying to spawn as often as possible every round.

coiax commented:
It's not guaranteed though, just highly likely.

MMMiracles commented:
i mean with no cost requirement it pretty much will be seen every round

ChangelingRain commented:
I'd like to note that right now the colossus spawns every round and is actually high enough chance that there will sometimes be two of them.
that said i'd like a better arena and maybe having the colossus move back to the arena if it loses aggro.

Xhuis commented:
I'll make a few changes based on the feedback I've received.

Xhuis commented:
_New in this commit:_
* Fixed some grammar issues and made the escaped prisoner flavor text more vague.
* Changed Legion's description and optimized his color flashing.
* Removed the watcher changes.
* Made ash walker spawn threshold a define.

Xhuis commented:
_New in this commit:_
* Added a `boldwarning` span class, which is exactly what it sounds like.
* Moved ash storm sound effects to a new folder, `sound/lavaland`.
* Players that fall through chasms are now stunned when they land.
* Anything that falls through surface chasms now has an animation as they die (link coming soon).
* Lots of grammar and messages have been changed.
* `lavaland_ruin_code.dm` has been mostly split into separate files for organization purposes.
* Sanguine sigils have a new [sprite](https://camo.githubusercontent.com/81559a78277c2f7a32cb9484d243e9f911feb9c5/687474703a2f2f692e696d6775722e636f6d2f55584b4b5778662e706e67).
* Cursed slot machines now have a 100% win chance if their user pulls it one last time before being gibbed, to give the illusion that they would have won had they lived.
* Cursed slot machines now spawn a bag of "money" instead of directly spawning a d20 of fate. The bag vanishes after one minute and can be activated with an empty hand to get your dice.
* Cursed slot machines now have sounds.
 * [Activate ](https://dl.dropboxusercontent.com/u/116194515/cursed_slot_machine.ogg)| [Jackpot](https://dl.dropboxusercontent.com/u/116194515/cursed_slot_machine_jackpot.ogg)

Commit:
Update to latest mater

Xhuis commented on _maps/map_files/generic/lavaland.dmm:
```diff
@@ -2080,16 +2080,9 @@
 	},
 /area/mine/living_quarters)
 "en" = (
-/obj/structure/rack,
```
Not sure why this stuff was removed. I repathed `lavaland_door` into `necropolis_gate` and that was the only thing I changed.

KorPhaeron commented:
>Cursed slot machines now have a 100% win chance if their user pulls it one last time before being gibbed, to give the illusion that they would have won had they lived.

Sounds abuseable with the number of throwaway lives running about on lavaland

Xhuis commented:
That's why I made it so that it only lasts for a minute. I could remove it if you'd like, though.

KorPhaeron commented:
Considering it can have catastrophic consequences for the round yes please

Commit:
Removed repathing and slot machine shenanigans

Xhuis commented:
Done. I also removed the map edits and gate repath because the map merger seems to be having problems and I can't get it to work properly.

Iamgoofball commented:
@Xhuis dont remove it, instead rework it to FAKE winning

Iamgoofball commented:
also u forgot to remove map edits

Commit:
Legion spawn changes

Xhuis commented:
I didn't forget - Travis just had a problem on its side.

Commit:
Map edits, round II

Xhuis commented:
Map edits are made again - all I really did was move the necropolis gate to the spot where it properly blocks light.

Core0verload commented on code/game/objects/structures/ghost_role_spawners.dm:
```diff
@@ -0,0 +1,235 @@
+		if(2)
+			flavour_text += "you're an exile from the Tiger Cooperative. Their technological fanaticism drove you to question the power and beliefs of the Exolitics, and they saw you as a \
+			heretic and subjected you to hours of horrible torture. You were hours away from execution when a high-ranking friend of yours in the Cooperative managed to secure you a pod, \
+			scrambled its destination's coordinates, and launched it. You awoke from stasis when you landed and have been surviving - barely - ever since.</b>"
```
Is it from that shitty Syndicate lore that is, for some reason, still in code?

Xhuis replied:
It sure is.

Core0verload replied:
I hoped it was abandoned forever.

Xhuis replied:
Too bad.

KorPhaeron commented:
WJ is in the process of remapping the entire thing right now so this might conflict

Xhuis commented:
If it's for the greater good, don't merge this.

KorPhaeron commented:
I like most of the rest of it though, you should just hold off on the path change of the door for now

Xhuis commented:
I did hold on off that - unless I missed it, it's not in anymore and is still under `lavaland_door`.

optimumtact commented:
>I would like to be the one to merge this in the end

Good luck

![Marko from taken, who says good luck in the movie to Liam Neeson](http://vignette3.wikia.nocookie.net/villains/images/5/57/Marko_the_kidnapper.png/revision/latest?cb=20130327144022)

Man-Shroom- commented on code/game/objects/structures/ghost_role_spawners.dm:
```diff
@@ -0,0 +1,235 @@
+			shoes = /obj/item/clothing/shoes/sneakers/black
+			back = /obj/item/weapon/storage/backpack
+		if(2)
+			flavour_text += "you're an exile from the Tiger Cooperative. Their technological fanaticism drove you to question the power and beliefs of the Exolitics, and they saw you as a \
```
This is all fine except for the shit about Exolitics, what the fuck is an Exolitic it's not in the code anywhere

Core0verload replied:
This lore still sucks.

Xhuis commented:
If the path change was the only thing you were concerned about, @KorPhaeron, this should be merge-ready.

KorPhaeron commented:
Give me a chance to read it all

Xhuis commented:
Jesus Christ that was fast. Mmkay, no problem.

optimumtact commented:
Enjoy your Kor approved stay on the review queue, it's FOR LIFE

KorPhaeron commented:
I like to read the things I merge

KorPhaeron commented on _maps/RandomRuins/LavaRuins/lavaland_surface_colossus_arena.dmm:
```diff
@@ -0,0 +1,44 @@
+"a" = (/turf/closed/mineral/volcanic/lava_land_surface,/area/ruin/powered{name = "Ominous Lair"})
```
I still dislike this sort thing but maybe I am crazy letting them roam about like they are

KorPhaeron commented on _maps/map_files/generic/lavaland.dmm:
```diff
@@ -2080,16 +2080,9 @@
-	baseturf = /turf/open/floor/plating/lava/smooth/lava_land_surface
-	},
-/area/mine/living_quarters)
+/obj/structure/lavaland_door,
```
Thought you were removing lavaland map edits

Xhuis replied:
I forgot to mention that. sorry! The necropolis gate's position before the edit was actually on a tile to the left of the hallway, making the weather control machine visible and making it impossible to knock on the door if you were to its right but possible to if you were two tiles to its left. I did some variable manipulation and properly centered it in the hallway.

KorPhaeron commented on code/game/objects/structures/ghost_role_spawners.dm:
```diff
@@ -0,0 +1,235 @@
+
+/obj/effect/mob_spawn/human/ash_walker/special(mob/living/new_spawn)
+	new_spawn.real_name = random_unique_lizard_name(gender)
+	new_spawn << "<b>Drag the corpses of men and beasts to your nest. It will absorb them to create more of your kind. Glory to the Dark One!</b>"
```
And here

KorPhaeron commented on code/modules/mining/lavaland/weather.dm:
```diff
@@ -46,7 +50,9 @@
-			M << "<span class='danger'><B>[duration_message]</B></span>"
+			M << "<span class='userdanger'><i>[duration_message]</i></span>"
+			if(duration_sound)
+				M << duration_sound
```
Have you tested how this sounds in game?

What happens if the sound plays and they leave the area?

Xhuis replied:
I've tested all the weather sounds. All of them play normally. If they're in a ruin or area where ash storms can't reach, they won't hear anything or see any messages. If they leave the area, the sound will continue, but they're all short enough that it could be viewed as simply leaving the storm for the shelter of another area.

KorPhaeron commented on code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm:
```diff
@@ -43,7 +43,10 @@
+			if(prob(50))
+				double_spiral()
+			else
+				seeking_laser()
```
Lets not do this. Delete the seeking laser code instead please

KorPhaeron commented on code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm:
```diff
@@ -4,7 +4,7 @@
 	icon_state = "legion"
 	icon_living = "legion"
-	desc = "One of many."
+	desc = "The Dark One from the fallen city. One of many."
```
Having all their desc exactly the same was part of the theme

KorPhaeron commented on code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm:
```diff
@@ -81,7 +81,7 @@
 		L.target = target
 
-		visible_message("<span class='danger'>[src] splits!</span>")
+		visible_message("<span class='boldannounce'>[src] splits in twain!</span>")
```
>twain

Xhuis replied:
vOv

KorPhaeron commented on code/modules/mob/living/simple_animal/hostile/mining_mobs.dm:
```diff
@@ -1007,14 +1007,14 @@
 	..()
-	visible_message("<B><span class='danger'>The tendril writhes in pain and anger and the earth around it begins to split! Get back!</span></B>")
-	visible_message("<span class='danger'>A chest falls clear of the tendril!</span>")
+	visible_message("<span class='boldannounce'>The tendril writhes in fury as the earth around it begins to crack and break apart!</span>")
```
I thought the explicit "get back!" was kind of important considering it's instant death

KorPhaeron commented on code/modules/ruins/objects_and_mobs/sin_ruins.dm:
```diff
@@ -0,0 +1,126 @@
+			H.visible_message("<span class='warning'>[H] pushes through [src]!</span>", "<span class='notice'>You've seen and eaten worse than this.</span>")
+			return 1
+		else
+			H << "<span class='warning'>You're repulsed by even looking at [src]. There's no way you can bring yourself to touch it.</span>"
```
You kinda removed the hint from how to pass it

KorPhaeron commented:
I guess I like about half of it. Mostly the mob spawner half of it.

Commit:
Some changes

Xhuis commented:
__New in this commit:__
* Removed all references to the Dark One.
* Reset Legion's description.
* Removed the disintegration ball and all code related to it.
* Re-added hints for the Necropolis tendril and gluttony's wall.

Commit:
Removes map changes, hopefully

KorPhaeron commented:
swear I will read this tonight

Commit:
Conflicts 1

Commit:
Updates the maps

Xhuis commented:
Oh, apparently fixing conflicts doesn't _actually_ fix the conflicts. Silly me.

Commit:
Conflicts 2

Commit:
Conflicts 3

Commit:
How about now?

Xhuis commented:
_New in this commit:_
* Fixed conflicts.
* Updated the animal hospital. Doors have been renamed to show the rooms they're going into, access requirements have been removed from the animal sleeper rooms, and a medical closet has been replaced with a biohazard suit closet.
* Remapped the [colossus arena](https://puu.sh/pu7P7/f073e59ef2.png). It now includes plenty of remains and a note on top of the sigil warning away the reader. A literal translation can be found below.

THE ANCIENT GUARDIAN WATCHES OVER THIS PLACE
WOE TO THE INVADER THAT SEEKS ENTRY TO THE NECROPOLIS
SPEAK THE WORDS OF THE SIGIL TO ISSUE YOUR CHALLENGE
AND FALL AS ALL OTHERS HAVE FALLEN.

Commit:
Remaps the colossus arena

Xhuis commented:
Another [remap ](http://i.imgur.com/QP7mIkW.png)of the colossus arena, thanks to @ChangelingRain's suggestions. The sanguine sigil now also shrinks instead of expanding.

ChangelingRain commented:
I'd still suggest having a busted-in wall and basalt around it, that appears to have lava around it.

Xhuis commented:
Wasn't sure how I wanted to do it with the remap. I decided on a "sealed away forever" theme for it and made the walls indestructible, so it's probably going to be pretty much pristine in the final push.

ChangelingRain commented on _maps/RandomRuins/LavaRuins/lavaland_surface_colossus_arena.dmm:
```diff
@@ -0,0 +1,28 @@
+"a" = (/turf/closed/indestructible{desc = "An extremely sturdy wall made of frigid metal, engraved with strange markings."; icon = 'icons/turf/walls/cult_wall.dmi'; icon_state = "cult"; name = "runed wall"},/area/ruin/powered)
```
Nooooooo. If you want indestructible cult walls, make a subtype of indestructible walls that actually smooths, don't edit the turfs.

Xhuis replied:
___YOU'RE NOT MY REAL DAD___
ok

KorPhaeron replied:
Use the necro turfs

Core0verload commented:
Can't wait for this to get merged. The only thing I don't like in it is shitty Tiger Corp lore, everything else is awesome.

Xhuis commented:
@Core0verload If it's any comfort, it's rarely going to be used and there's no in-game evidence of their existence other than that paragraph.

Commit:
Restores old colossus spawning

Xhuis commented:
Removed colossus arena and replaced it with regular spawning, as requested by @KorPhaeron.

KorPhaeron commented on code/modules/ruins/lavaland_ruin_code.dm:
```diff
@@ -75,364 +43,3 @@
-
-//Free Golems
-
-/obj/item/weapon/disk/design_disk/golem_shell
```
Why did you delete this?

Please stop moving shit into new files FNR, it makes it hard to keep track of changes

KorPhaeron commented on code/modules/ruins/objects_and_mobs/ash_walker_den.dm:
```diff
@@ -0,0 +1,39 @@
+#define ASH_WALKER_SPAWN_THRESHOLD 2
```
I'm not sure why this is in its own file either come to think of it, why scatter ash walker stuff across the codebase?

Xhuis replied:
`lavaland_run_code.dm` is an ambiguous name and can imply several things. I split each unique ruin's specific objects into different files to be more organized, and ash walker eggs aren't in this file because I put them into `ghost_role_spawners.dm`.

KorPhaeron commented on code/modules/mining/mine_turfs.dm:
```diff
@@ -625,7 +627,20 @@
+			L.notransform = TRUE
+			L.Stun(10)
+			L.resting = TRUE
+		animate(AM, transform = matrix() - matrix(), alpha = 0, color = rgb(0, 0, 0), time = 10)
```
@Xhuis 

it was you after all #19500

Xhuis replied:
Damn! I didn't know that `animate()` had sleeps in it. Sorry...
